lib/plugins/aws/package/compile/events/websockets/lib/stage.js

```diff
@@ -28,9 +28,6 @@ module.exports = {
         Type: 'AWS::ApiGatewayV2::Stage',
         Properties: {
           ApiId: this.provider.getApiGatewayWebsocketApiId(),
-          DeploymentId: {
```

Comment:
Let's add comment here that `DeploymentId` is configured in `deployment.js`

Reply:
Good idea. Added